src/property-filter/property-editor.tsx

```diff
@@ -43,6 +57,55 @@ export function PropertyEditorContent<TokenValue = any>({
   );
 }
 
+export function PropertyEditorContentEnum({
+  property,
+  filter,
+  value: unknownValue,
+  onChange,
+  asyncProps,
+  filteringOptions,
+  onLoadItems,
+}: {
+  property: InternalFilteringProperty;
+  filter: string;
+  value: null | string[];
+  onChange: (value: null | string[]) => void;
+  asyncProps: DropdownStatusProps;
+  filteringOptions: readonly InternalFilteringOption[];
+  onLoadItems?: NonCancelableEventHandler<LoadItemsDetail>;
+}) {
+  const valueOptions: readonly { value: string; label: string }[] = filteringOptions
+    .filter(option => option.property?.propertyKey === property.propertyKey)
+    .map(({ label, value }) => ({ label, value }));
+  const valueHandlers = useLoadItems(onLoadItems, '', property.externalProperty);
+  const value = !unknownValue ? [] : Array.isArray(unknownValue) ? unknownValue : [unknownValue];
+  const selectedOptions = valueOptions.filter(option => value.includes(option.value));
+  const filteredOptions = filterOptions(valueOptions, filter);
+  return (
+    <div className={clsx(styles['property-editor'], styles['property-editor-enum'])}>
+      {filteredOptions.length === 0 && (
+        <div className={styles['property-editor-header-enum']}>
+          <InternalCheckbox checked={false} readOnly={true} />
+          {property.groupValuesLabel}
+        </div>
+      )}
```

Comment:
Why is this section not already handled by the embedded multiselect?

Reply:
Very good question!

We need to ensure the group option always exists - that's why it is excluded from filtering and then inserted separately. However, that does not work well in case there are no options or no matched options. In such case the group text is greyed out not meeting color contrast and the empty text is not shown (because the group option still counts). 

<img width="576" alt="Screenshot 2024-10-21 at 10 53 41" src="https://github.com/user-attachments/assets/f6d8a291-f62b-453c-908c-2fc51b7a2e32">

Reply:
OK. Can we have a visual regression test for this case?

Reply:
Will do!

There is a test page already: /light/property-filter/token-editor, and I will add a screenshot test.